modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/am/integration/test/ClientAuthenticator.java

```diff
@@ -67,11 +68,15 @@ public static String getAccessToken(String scopeList, String appName, String cal
             String clientEncoded = DatatypeConverter.printBase64Binary(
                     (consumerKey + ':' + consumerSecret).getBytes(StandardCharsets.UTF_8));
             urlConn.setRequestProperty("Authorization", "Basic " + clientEncoded);
-            if (!"carbon.super".equalsIgnoreCase(tenantDomain)) {
+            if (!"carbon.super".equalsIgnoreCase(tenantDomain) && !username.contains("@")) {
```

Comment:
Shall we use the constants here?

Reply:
Fixed in 09e067607b550f97373afb9510ea71424bed8a67